live-build/variants/internal-dcenter/package-lists/dcenter.list.chroot

```diff
@@ -26,7 +26,7 @@ nfs-common-dbgsym
 nfs-kernel-server
 nfs-kernel-server-dbgsym
 openjdk-11-jdk-headless
-python3
+python3.12
 python3-dbg
 python3-dev
 python3-venv
```

Comment:
I think these three packages also need the python version bumped so that we are installing the python3.12 compatible versions of the packages.